libraries/chain/include/eosio/chain/exceptions.hpp

```diff
@@ -106,6 +106,12 @@
 
 namespace eosio { namespace chain {
 
+   enum class system_error_code : uint64_t {
+      default_system_error = 10000000000000000000ULL,
```

Comment:
`default_system_error` seems like an odd name to me. Maybe `reserved_system_error` or `lowest_reserved_system_error` or `system_error_range` or `system_error_range_begin`